src/Symfony/Bundle/FrameworkBundle/Routing/Router.php

```diff
@@ -173,4 +175,14 @@ private function resolve($value)
 
         return str_replace('%%', '%', $escapedValue);
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    public static function getSubscribedServices()
+    {
+        return array(
+            'routing.loader' => LoaderInterface::class,
+        );
+    }
```

Comment:
@nicolas-grekas The service definition has not been updated (not tagged `container.service_subscriber`) so this is actually unused, is it expected?